src/platform/user/authentication/components/NewDesignButtons.jsx

```diff
@@ -52,7 +49,8 @@ export default function NewDesignButtons({
           type="button"
           aria-label="Sign in with Login.gov"
           className="usa-button logingov-button vads-u-margin-y--1p5 vads-u-padding-y--2"
-          onClick={() => loginHandler('logingov')}
+          data-csp={CSP_IDS.LOGIN_GOV}
```

Comment:
Data tag added for ease of testing. Easier to discern which button is which without relying on parsing the `className`. Parsing the classname is also not fool-proof as it can change.